graphqlmetrics/pkg/telemetry/tracing.go

```diff
@@ -0,0 +1,70 @@
+package telemetry
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+)
+
+const (
+	// WithEndpoint respects the following environment variables:
+	// OTEL_EXPORTER_OTLP_ENDPOINT
+	// OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+	DefaultGrpcTelemetryEndpoint = "cosmo-otel.wundergraph.com"
+)
+
+func buildResourceOptions() resource.Option {
```

Comment:
We don't have to duplicate those information when they are already cover above with standardized attributes.

Reply:
I would rather remove the tracing related implementations for now and bring them in, when tracing is integrated.